### DISCUSSION TITLE
npm run login fails to send email

### DISCUSSION THREAD

**Discussion by MegaMotion** [mozilla/hubs#3032] [General Questions & Discussion] [answered]
Setting up a new stack, everything is fine up to the point of trying to install my client, but when I run login and give it my stack URL and my admin email, it never sends the email. Anybody got any hints for debugging this?

**Comment by robinkwilson** [comment, marked as answer]
Did you successfully login to your HC instance admin panel? + see data populate in Server Settings (to check your email is an admin)?
Did you check your Spam folder?

**Reply by MegaMotion** [reply]
Yes, and yes. This time. ;-)

**Reply by MegaMotion** [reply]
Ah, but this morning it worked! I swear I checked my spam folder yesterday, but it appears that those emails ultimately made it in there, so... let's call this one resolved due to user error. :-) Thank you!
